ai/github_issue_handling/issue_clawer.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
This PR is only for gpu dockefile, we can move filter script change to https://github.com/XinShuYang/antrea/pull/5/files @cntnfire @Createsnow

Reply:
It seems that my commit history has also been re-submitted. Perhaps @cntnfire should commit only the specified modified files?

Reply:
@Createsnow  For all filter related commits, you should post them in this PR #5 . This PR #10 is only for GPU dockerfile